jetty-server/src/main/config/etc/jetty-inetaccess.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "http://www.eclipse.org/jetty/configure_9_3.dtd">
+
+<Configure id="Server" class="org.eclipse.jetty.server.Server">
+    <Call name="insertHandler">
+        <Arg>
+            <New id="InetAccessHandler" class="org.eclipse.jetty.server.handler.InetAccessHandler">
+                <Call name="include"><Arg>127.0.0.1-127.0.0.255</Arg></Call>
```

Comment:
This file should not have specific IPs or names specified.
Perhaps you could comment it out?

Reply:
i think this is a copy paste from the jetty 10 fyi. but yes will do.

Reply:
Could also have a property with static calls to String.split that will turn it into an array that can be passed to include.